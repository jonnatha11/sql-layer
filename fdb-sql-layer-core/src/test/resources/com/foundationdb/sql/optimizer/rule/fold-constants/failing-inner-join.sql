SELECT name, order_date, sku
  FROM customers 
 INNER JOIN orders ON customers.cid = orders.cid
 INNER JOIN items ON orders.oid = items.oid AND 1=0
