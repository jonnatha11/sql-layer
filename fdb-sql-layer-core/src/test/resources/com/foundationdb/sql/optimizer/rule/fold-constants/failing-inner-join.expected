SelectQuery@6e4eeaaf
  ResultSet@a21d23b[name, order_date, sku]
    Project@327124af[NULL, NULL, NULL]
      NullSource@664310d0
